src/CSET/operators/read.py

```diff
@@ -252,6 +253,27 @@ def _lfric_normalise_callback(cube: iris.cube.Cube, field, filename):
         cube.attributes["um_stash_source"] = str(sorted(ast.literal_eval(stash_list)))
 
 
+def _lfric_time_coord_fix_callback(cube: iris.cube.Cube, field, filename):
```

Comment:
Coding appropriate for creating DimCoord from assumes time_coord as AuxCoord.

Worth reviewing alignment with https://code.metoffice.gov.uk/trac/rmedtoolbox/browser/main/branches/dev/huwlewis/r637_res_um_lfric/lib/lfric.py?rev=640#L51 as example of similar processing as inherited from "Paul Earnshaw scripts" - part of the 'UM-ify' and introduced as RES/Toolbox callback. 

In particular, 
i) that approach uses iris.util.promote_aux_coord_to_dim_coord(),
ii) do we need to add any processing to convert units to hours (or is that handled elsewhere)
iii) do we need to remove or adjust bounds?

Reply:
[iris.util.promote_aux_coord_to_dim_coord](https://scitools-iris.readthedocs.io/en/stable/generated/api/iris.util.html#iris.util.promote_aux_coord_to_dim_coord) looks perfect. I'll switch to using that.

I have yet to encounter any files that don't use hours, and time processing in CSET works with datetime objects. So as long as it is a valid cf_units time unit, it should work fine whatever it is.

Bounds are an interesting one. I'm having to add them in the pcolormesh plotting operator, but it might be better off being done at load time. I can't think of anything that iris.coords.Coord.guess_bounds should break.